diskmaker/controllers/lv/reconcile.go

```diff
@@ -59,6 +60,14 @@ const (
 	ComponentName = "localvolume-symlink-controller"
 )
 
+var watchNamespace string
+var nodeName string
+
+func init() {
+	nodeName = common.GetNodeNameEnvVar()
+	watchNamespace, _ = common.GetWatchNamespace()
```

Comment:
Where are we using this?

Reply:
Not using the watchNamespace in LV controller. Removed.